lib/wallaby/browser.ex

```diff
@@ -200,16 +200,16 @@ defmodule Wallaby.Browser do
   Screenshots are saved to a "screenshots" directory in the same directory the
   tests are run in.
 
-  Pass %{name: "some_name"} to specify the file name. Defaults to a timestamp.
+  Pass `[{:name, "some_name"}]` to specify the file name. Defaults to a timestamp.
   """
   @spec take_screenshot(parent) :: parent
 
-  def take_screenshot(%{driver: driver} = screenshotable, opts \\ %{}) do
+  def take_screenshot(%{driver: driver} = screenshotable, opts \\ []) do
     image_data =
       screenshotable
       |> driver.take_screenshot
 
-    path = path_for_screenshot(%{name: opts[:name]})
+    path = path_for_screenshot(name: opts[:name])
```

Comment:
What do you think about writing it like this?

```elixir
name = Keyword.get(opts, :name, :erlang.system_time)
path = path_for_screenshot(name)
```

That should simplify `path_for_screenshot/1` to

```elixir
defp path_for_screenshot(name) do
  File.mkdir_p!(screenshot_dir())
  "#{screenshot_dir()}/#{name}.png"
end